### PR TITLE
add some missing import in client

### PR DESCRIPTION
**Pull request by joejoe321321**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by chinkong83** [comment]
hi, @joejoe321321 , please modify testdata/SampleClient.txt or testdata/SampleClientImpl.txt for test

File: testdata/SampleClientImpl.txt [hunk 1]
```diff
@@ -18,12 +19,11 @@ import com.fasterxml.jackson.databind.ObjectMapper;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.ws.rs.core.Response;
 import javax.ws.rs.core.UriBuilder;
 import java.net.URI;
 import java.util.HashSet;
```
Comment (geekybaiyi) [inline testdata/SampleClientImpl.txt:24]: so looks like importing HasSet is optional and this test case tested when  it is true, do we need another test when it is false?

Comment (joejoe321321) [reply testdata/SampleClientImpl.txt:24]: OK

**Pull request merged**